TERMSOFSERVICE.md

```diff
@@ -69,19 +66,64 @@ by means of an individual agreement.
 reasons. If a registration is incomplete, we are entitled to delete the user
 account within one week.
 
-# 4. Account, password and security
+# 4. Account, password and security, Requirements for Usage, Privacy, Sanctions and Blocking of personal Accounts
 
 4.1. Registering and using Tideways may involve you setting up an account
 and giving a password.
 
-4.2. You are fully responsible for maintaining the confidentiality of your
-account and the password, and all activities that occur under your account or
-password. You agree to immediately notify us of any unauthorized use of your
+4.2. Consumers may not register based on the preceding section 4.1.. A
```

Comment:
Clarify that consumers are not allowed to register for Tideways and that an account is created by someone with permission by a lawful representative.